packages/tradershub/src/features/cfd/screens/CFDCompareAccounts/CompareAccounts.classnames.ts

```diff
@@ -0,0 +1,18 @@
+import { cva, VariantProps } from 'class-variance-authority';
+import { ExcludeAllNull } from '@deriv/quill-design';
+
+export const CompareAccountsPlatformLabelClass = cva('bg-system-light-platform-background p-[9px] rounded-t-700', {
+    variants: { background: { CTrader: 'bg-[#ffeabf]', MT5: 'bg-[#e6f5ff]', OtherCFDs: 'bg-[#e8fdf8]' } },
+});
+
+export const CompareAccountsPlatformLabelTextColorClass = cva('', {
+    variants: { label: { CTrader: 'text-[#ff9c13]', MT5: 'text-[#2C9aff]', OtherCFDs: 'text-[#17eabd]' } },
+});
```

Comment:
Can move into one component but the problem is the main class of this two different divs.

Reply:
Added custom colors as they are only used here only. So its better to use it in CVA to have a cleaner version instead of tweaking tailwind.config

Reply:
To be honest extending the colors  `tailwind.config.js` is not a really bad idea although it's being used in one file at the moment. Maybe put it under the a section called `cfd` maybe or extend whatever we have in the `brand` .

Reply:
Will try to add a new section or extend random for colors